src/Utilities/ToolTask.cs

```diff
@@ -658,6 +658,17 @@ private string GetTemporaryResponseFile(string responseFileCommands, out string
             return startInfo;
         }
 
+        /// <summary>
+        /// Starts the process during task execution
+        /// </summary>
+        /// <param name="proc"></param>
+        /// <returns></returns>
```

Comment:
Please add good doc comments. The more, the better, as this will end up on learn.microsoft.com as public API documentation.